### DISCUSSION TITLE
Is this project still alive?

### DISCUSSION THREAD

**Discussion by nootkan** [ryanhowdy/fcms#629] [Q&A] [answered]
Hi Ryan, been a long time since I visited your cms.

I still have one running but the family has pretty much stopped using it with the exception of my father and his side of the family.

So I thought I'd revisit the project and upgrade it or start over for him as he enjoyed being able to keep up with his nieces and nephews that live so far away.

Is there a newer version than 3.7?  I cannot seem to install without errors it seems.

Tried resolving them myself by googling the errors but can't get past step 5 due to a Notice:  

Notice: Undefined variable: mysqli on \inc\install_inc.php on line 204

and 

Fatal error: Uncaught Error: Call to a member function query() on null  \inc\install_inc.php on line 204

here is line 204:

`$connection->query($sql) or die($sql . '<br/>' . $connection->error());`

Thanks for keeping this great script alive and hopefully you and your family are well and things are going well for you.

**Comment by gOOvER** [comment]
The lastest Commit is from Aug 13, 2018; 4 years ago. So i think, this project is dead

**Comment by ryanhowdy** [comment, marked as answer]
Well I'd like to think that one day I'll get back to this project, but it isn't looking good.

There are several reasons I haven't worked on it lately....

- Unfortunately my family stopped using the software a while ago, and that was my main driver in working on it.
- I've also moved on to several other pet projects (most of them are private repos, but maybe i'll make them public one day).
- The codebase is old and crappy, mostly written before I knew what i was doing.  If i ever restart the project, it will probably start with me rewritting it in Laravel or something a little more modern.
- And last, I'm just not sure there is enough interest in a project like this.  Apparently people would rather just use facebook


Please see this announcement for more information about the [future of Family Connecitons](https://github.com/ryanhowdy/fcms/discussions/635).

**Reply by ebretteville** [reply]
I do hope to use it in the futur, as facebook is more and more crappy with datas...

**Reply by fermino** [reply]
If at any point you'd like to return to the project and find useful a couple of hands to help, I'm more than willing to do so. I've used the project for a couple of years and I'm quite fond of it :)

**Reply by ryanhowdy** [reply]
Thanks fermino.  I am going to spend the next few weeks trying to figure out a way to restart the development of the site.  When I get everything figured out, i'll reach out.

**Reply by CptHowdy2012** [reply]
@ryanhowdy any luck in resuming work on this?. . . id love to have a working copy to try to get my family to contribute to its use, since facebook has become the ultimate data collection tool.

edit:
after i posted this, i found the tracker for 4.0 .  anxious to do some testing with that when its ready. btw I think the docker image will be the best answer for deployment given the change in framework.

**Reply by ryanhowdy** [reply]
I am pretty close to what I'm calling 4.0 alpha.  Which should include most of the basic functionality.  This will allow users to set it up and test and see how things are going to work going forward.

**Comment by CptHowdy2012** [comment]
i been away for a bit. . . . any new progress lately?

**Comment by ebretteville** [comment]
Hye, On my webhost provider I can use softaculous to install laravel, two versions are available: 8.83.27 and 9.52.2, which one do I need to choose to be able to use familyconnection 4.0.0? Thanks

**Reply by ryanhowdy** [reply]
Currently the minimum requirement is Laravel 9.0.  So 9.52.2 should work just fine.

**Reply by ebretteville** [reply]
Thank you

**Comment by leolivier** [comment]
Hi Ryan, 
FCMS 4 is definitely what I was looking for since years!
But always the same question: is the project still alive? There's been no commit on 4.0.0 branch since April 11...
I created a damn simple docker image and setup of the alpha version of fcms in very few hours and I'll make a PR for that soon if this is alive...
(EDIT: and ready to help more if needed but I'm not at all a Laravel expert)

**Reply by ryanhowdy** [reply]
Yea, it's still alive and I am currently working on the calendar invitation system.  I have a big bunch of changes to be checked in.

The problem as always is that I have a full time job that keeps me busy.

**Comment by leolivier** [comment]
Hi Ryan, good news.
I worked hundreds of hours on an open source project which was finally abandoned due to lack of availability of other developers, so I 'm a bit cautious with that now, no offense...
I'll do a PR next weekend for the docker thing.
